ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -512,6 +508,8 @@ def plot_LoD(
         [item for sublist in filters_list for item in sublist if item in df_mrd_sim.index.values], c_lod
     ].min()  # best LoD across all plotted results
 
+    df_mrd_sim.to_parquet(pjoin(os.path.dirname(output_filename), "df_mrd_sim.parquet"))
```

Comment:
This is already saved in the end of the retention_noise_and_mrd_lod_simulation function, remove please.

Reply:
done